src/conversion_helper.js

```diff
@@ -724,11 +737,17 @@ function findPotentialConversions(comment) {
 function filterConversions(potentialConversions) {
   const possiblyValidConversions = potentialConversions.filter(input => {
     const imperialUnit = input['imperial']['unit'];
-    const imperialNumber = Number(input['imperial']['number']);
+    const imperialNumber = input['imperial']['number'];
 
     const map = unitLookupMap[imperialUnit];
     if (map['isInvalidInput']) {
-      return map['isInvalidInput'](imperialNumber) == false;
+      let result = false;
+      imperialNumber.forEach(function(item) {
+        if (!map['isInvalidInput'](Number(item))) {
+          result = true;
+        }
+      });
+      return result;
```

Comment:
In the old behaviour, this boolean true/false referred to one single number. For the string "-10 miles 20 miles" it would return false for -10 and true for 20, which means only 20 miles will be converted

Because we refactored this into an array, if it has the inputs [-10, 20] ... it would return a single boolean "true" which would allow both -10 and 20 to be converted. Am I interpreting the code correctly? If so, this might be a problem (and tests should be failing!!! Maybe this case wasn't tested?)

Reply:
Oops, don't know what I thought while writing this.
Fixing.